courses/models_test.py

```diff
@@ -286,3 +334,17 @@ def test_course_page_properties():
         assert block.value == "benefit"
     assert course.outcomes == learning_outcomes_page
     assert not LearningOutcomesPage.can_create_at(course_page)
+
+    # ForTeamsPage tests
```

Comment:
same here, 

or we can combine tests for subtypes for product (course/program) and separate them based on subtype.